unpackaged/config/trial_tso/prompts/ReportGuidance.prompt

```diff
@@ -4,7 +4,7 @@
     <promptVersions>
         <actionButtonLabel>Learn About Reports</actionButtonLabel>
         <actionButtonLink>https://sfdc.co/QVnrG</actionButtonLink>
-        <body>&lt;p&gt;Bring your data to life by leveraging Reports. NPSP comes with a number of reports you can take advantage of immediately. &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;To get started, select ‘All Reports’ from the menu on the left to see which reports are available to you. Learn more about these specific reports in the &lt;a href=&quot;https://powerofus.force.com/s/article/NPSP-Reports-Workbook&quot; rel=&quot;noopener noreferrer&quot; target=&quot;_blank&quot;&gt;Power of Us Hub&lt;/a&gt;. &lt;/p&gt;</body>
+        <body>&lt;p&gt;Bring your data to life by leveraging Reports. NPSP comes with a number of reports you can take advantage of immediately. &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;To get started, select ‘All Reports’ from the menu on the left to see which reports are available to you.</body>
```

Comment:
Couldn't we link here for this one, or is this out of date and not worth using? https://help.salesforce.com/s/articleView?id=sfdo.NPSP_Reports_Workbook.htm&type=5

Reply:
That looks quite similar, but would wanna make sure to defer to @ErinWiedemer on this!

Reply:
HI @balsamhindi, I'm a little torn on this one.
I was just in a meeting yesterday with a communications writer about this Reports Workbook. It is out of date, but she's going to completely re-write it. So I think it is okay to add Luke's link. However, we're trying to get away from posting links in the product, so that if the links change, you don't have to go through all of this work again. 

Let's stick with the original plan and not include it. I think that we're going to develop a trail on NPSP reports, and that will be a lot more helpful than this link anyway.